src/components/TaskTracker/Settings.tsx

```diff
@@ -62,23 +65,23 @@ export const Settings = ({ setOpenSettings, Task }) => {
           </div>
         </div>
       </div>
-      <div className="flex items-center justify-between border-b-2 border-gray-100 px-2 pb-2">
+      {/* <div className="flex items-center justify-between border-b-2 border-gray-100 px-2 pb-2">
```

Comment:
Left over commented code?
Reason we need to leave it?

Reply:
Left over code for sure. 

Resolved in c9b6320109809b08f43a9f577ed4c102ddc1146d